diode-proto/diode/v1/reconciler.proto

```diff
@@ -70,6 +70,12 @@ message IngestionMetrics {
   int32 no_changes = 5;
 }
 
+// A change set
+message ChangeSet {
+  string id = 1; // A change set ID
+  string data = 2; // Base64 string representing the change set
```

Comment:
Could we ament comment for `data`, please? I'd suggest to use something generic, as data can be of any string form (raw JSON string or brotli or something else), e.g.
```suggestion
  string data = 2; // A change set data
```